src/AccessibilityInsights.SharedUx/Controls/TestTabs/AutomatedChecksControl.xaml.cs

```diff
@@ -873,10 +867,6 @@ private void GroupItem_PreviewKeyDown(object sender, KeyEventArgs e)
                 {
                     exp.IsExpanded = true;
                 }
-                else
-                {
-                    urlLink?.Focus();
-                }
                 e.Handled = true;
             }
             else if (e.Key == Key.Left)
```

Comment:
we might be able to remove the code for going left from the hyperlink too (line 884)

Reply:
Removed this and the hyperlink launching code